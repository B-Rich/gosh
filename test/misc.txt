              Document Title

               Norman Feske


A paragraph without a chapter.

Abstract
########

Ohh.. .wie abstrakt....

Chapter One
###########

Some _accentuations with spaces in-between_

A paragraph with an
'apostroph' at the beginning of the line.



