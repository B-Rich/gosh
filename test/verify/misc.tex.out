\documentclass[11pt,ngerman,a4paper,normalheadings,DIV14]{scrartcl}
\usepackage{times}
\usepackage[T1]{fontenc}
\usepackage[latin1]{inputenc}
\usepackage[small,bf,hang]{caption2}
\usepackage[ngerman]{babel}
\usepackage{epsfig}
\emergencystretch = 10pt
\clubpenalty = 10000
\widowpenalty = 10000
\displaywidowpenalty = 10000
\usepackage{amsmath}
\usepackage{amssymb}
\begin{document}
\title{Document Title}
\author{Norman Feske}
\maketitle
 A paragraph without a chapter.


 %     -+*|[ ABSTRACT ]|*+-

 \begin{abstract} \label{Abstract}

  Ohh.. .wie abstrakt....

 \end{abstract}

 %     -+*|[ CHAPTER ONE ]|*+-

 \section{Chapter One} \label{Chapter One}

  Some \emph{accentuations with spaces in-between}

  A paragraph with an
  \texttt{apostroph} at the beginning of the line.



\newpage
\bibliographystyle{plain}
\bibliography{custom,master}
\end{document}
